pkg/controller/common/cleaner.go

```diff
@@ -23,7 +24,7 @@ import (
 )
 
 type cleaner struct {
-	name       string
+	name       controllername.ControllerNamer
```

Comment:
nit if the `name` field is a `namer` then does the name:`namer` return also a name? idk if changing this to a "namer" field is clearer or makes it more complex

Reply:
we could rename the field to controllername? or controllernamer? I think I'd prefer it be called "namer" rather than name since it's something that generates names for different situations

Reply:
going to resolve so i can merge and get a release out. namer is maybe a better name and can rename in a future pr after we hit this release deadline